### ISSUE TITLE
As a mug I want to see my priority tasks on a home screen

### ISSUE THREAD

**Issue by UglyAlex** [uglymugs/JustNUM#5]
Tasks would be prioritised using a traffic light system.

Red = NEW report submitted by an INDIVIDUAL member
Orange/amber = NEW report submitted by project
Green = all other reports

**Comment by UglyKerri** [comment]
I would see this as prioritising new reports and not priority tasks of reports that we have already started to work on.

**Comment by UglyAlex** [comment]
Yes, agreed.  But on the "outstanding tasks" area both new reports and unfinished tasks from others should appear together.  So, having new reports from individuals at the top of the list, followed by new reports from projects or police, followed by the reports we've already started working on.  That's what the above outlines.  It's not really a priority, just the order in which it appears on the screen.
